main.go

```diff
@@ -1,12 +1,33 @@
 package main
 
+import "errors"
+
 func Divine()  {
 	
 }
 
 func Add2()  {
 	
 }
+
+func returnErr() error {
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>func `returnErr` is unused (unused)